### PR TITLE
Move to Track 2 SDKs

### PR DESCRIPTION
**Pull request by alexweininger**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -211,12 +211,12 @@
...
     "devDependencies": {
         "@microsoft/eslint-config-azuretools": "^0.1.0",
-        "@microsoft/vscode-azext-dev": "^0.1.4",
+        "@microsoft/vscode-azext-dev": "^2.0.0",
         "@types/fs-extra": "^8.1.1",
         "@types/gulp": "^4.0.8",
         "@types/mocha": "^8.2.2",
         "@types/node": "^14.0.0",
-        "@types/vscode": "1.66.0",
+        "@types/vscode": "^1.78.0",
```
Comment (bwateratmsft) [inline package.json:219]:
```suggestion
        "@types/vscode": "1.78.0",
```

Comment (bwateratmsft) [reply package.json:219]: And do `npm i` once more after

File: src/utils/requestUtils.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+export function createRequestUrl(base: string, queryParams: Record<string, string>): string {
```
[the remark this answers is not in the document]
Comment (alexweininger) [reply src/utils/requestUtils.ts:6]: I want to wait to add it to utils until I see what we need in the other extensions. I think the other extensions might need some more complex options. Plus I would like to build this funcionality back into `sendRequestWithTimeout`.

**Review by nturinski — APPROVED** [review]
I'm sure that you're seeing the build error though :/

**Comment by alexweininger** [comment]
> I'm sure that you're seeing the build error though :/

Yeah, using Node 16 instead of 14 should fix it. Don't ask me why

**Pull request merged**